### PR TITLE
FLU: Convert Pioneer robots

### PR DESCRIPTION
**Pull request by lukicdarkoo**
- [x] (506) robots/adept/pioneer2/protos/Pioneer2.proto
- [x] (507) robots/adept/pioneer3/protos/Pioneer3Gripper.proto
- [x] (508) robots/adept/pioneer3/protos/Pioneer3dx.proto
- [x] (509) robots/adept/pioneer3/protos/Pioneer3at.proto
- [x] (510) robots/adept/pioneer3/protos/Pioneer3DistanceSensor.proto
- [x] (191) projects/robots/adept/pioneer2/worlds/pioneer2.wbt
- [x] (192) projects/robots/adept/pioneer3/worlds/pioneer3at.wbt
- [x] (193) projects/robots/adept/pioneer3/worlds/pioneer3dx_matlab.wbt
- [x] (194) projects/robots/adept/pioneer3/worlds/pioneer3dx_with_kinect.wbt
- [x] (195) projects/robots/adept/pioneer3/worlds/pioneer3dx.wbt
- [x] (196) projects/robots/adept/pioneer3/worlds/pioneer3dx_gripper.wbt
- [x] (197) projects/robots/adept/pioneer3/worlds/pioneer3dx_collision_avoidance.wbt

This PR also fixes a kinematic mode for FLU robots and deletes some leftovers from the DifferentialDrive node.

### CODE REVIEW

File: projects/robots/adept/pioneer3/worlds/pioneer3dx_gripper.wbt [hunk 1]
```diff
@@ -6,44 +6,46 @@ WorldInfo {
...
 DEF RED_TEE Solid {
-  translation -0.0376035 0 -0.618808
+  translation -0.0376035 0.618808 0
+  rotation 1 0 0 1.5708
   children [
     Group {
       children [
         DEF PITCH_PLATE Transform {
           translation 0 0.0465 0
+          rotation 0 1 0 0
```
Comment (BenjaminDeleze) [inline projects/robots/adept/pioneer3/worlds/pioneer3dx_gripper.wbt:48]:
```suggestion
```

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Thank you.
Only some minor changes:

- The two small cylinder bounding objects on top of the `Pioneer3at` seem wrong (it was already the case in R2021b, but I think we should correct it).
- The base translation of the `Pioneer3dx` needs to be converted.
- The initial orientation of the `Pioneer2` in `pioneer2.wbt` is different than before.
- The initial orientation of the `Pioneer3dx` in `pioneer3dx_collision:avoidance.wbt` is different than before (not important in my opinion)

Note that I could not really test the matlab world as I do not have matlab

**Comment by lukicdarkoo** [comment]
> * The initial orientation of the `Pioneer2` in `pioneer2.wbt` is different than before.
> * The initial orientation of the `Pioneer3dx` in `pioneer3dx_collision:avoidance.wbt` is different than before (not important in my opinion)

I changed these as I believe the new viewpoint is better.


> * The base translation of the `Pioneer3dx` needs to be converted.

Do you mean changing default translation or making the `0 0 0` translation to be above the ground?

**Comment by BenjaminDeleze** [comment]
> I changed these as I believe the new viewpoint is better (it depicts the robot's behavior better). Should I revert it?

No it is ok for me like that.



> Do you mean changing default translation or making the `0 0 0` translation to be above the ground?

I meant changing the default, so it is good now

**Review by BenjaminDeleze — APPROVED** [review]

**Pull request merged**